ground/src/main/java/com/google/android/ground/system/auth/GoogleAuthenticationManager.kt

```diff
@@ -91,11 +98,24 @@ constructor(
 
   override fun signIn() {
     signInState.onNext(SignInState.signingIn())
+    if (USE_EMULATORS) {
```

Comment:
Same as above

Reply:
Unless we expose this as a Provider, it would require us to call getInstance() earlier, during DI initialization, rather than on demand. I can try.